lib/lure/repo.go

```diff
@@ -85,12 +84,16 @@ func checkForUpdatesJob(auth Authentication, project Project) error {
 	err, modulesToAdd := mvnOutdated(repo.LocalPath() + "/" + project.BasePath)
 	modulesToUpdate = appendIfMissing(modulesToUpdate, modulesToAdd)
 	log.Printf("Modules to update : %q", modulesToUpdate)
-	pullRequests := getPullRequests(auth, project.Owner, project.Name)
+
+	ignoreDeclinedPRs := os.Getenv("IGNORE_DECLINED_PR") == "1"
+	pullRequests := getPullRequests(auth, project.Owner, project.Name, ignoreDeclinedPRs)
 
 	for _, moduleToUpdate := range modulesToUpdate {
 		updateModule(auth, moduleToUpdate, project, repo, pullRequests)
 	}
 
+	closeOldBranchesWithoutOpenPR(auth, project, repo)
```

Comment:
No need to do it now but it'd be cool to have this as a separate command as well, that I could configure to clean up old branches that match a regex and are more than X days old, and without a PR or something.

Reply:
Yup, agreed. The main reason I didn't is because we have this tool internally for hg, but I think we don't for git.